frontend/src/components/rota/index.tsx

```diff
@@ -1,10 +1,58 @@
 import React from 'react';
+import FullCalendar from '@fullcalendar/react';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import tippy from 'tippy.js';
 import rotaStyle from './rota.module.scss';
+import 'tippy.js/dist/tippy.css';
+import '@fullcalendar/core/main.css';
+import '@fullcalendar/daygrid/main.css';
+import '@fullcalendar/timegrid/main.css';
+
+const DUMMY_EVENTS = [
+  {
+    start: '2020-05-08T08:39:14Z',
+    end: '2020-05-08T19:41:14Z',
+    title: 'slacking off plz pay',
+    description: 'no, seriously, I need that money',
+    color: 'seagreen',
+  },
+  {
+    start: '2020-05-08T11:28Z',
+    end: '2020-05-08T13:33Z',
+    title: 'lunch',
+    description: 'FOOOD',
+    color: 'gold',
+  },
+];
+
+const tooltipFunction = (info: any) => {
```

Comment:
is there a way of specifying what happens if the description is empty/undefined?
But actually, should we even worry about it?

Reply:
We'll be the ones choosing what to put in the description (I think it's name, surname, times and engagement type?), so I wouldn't worry about it at this stage.